Assets/Scripts/Boat/BoatCapacity.cs

```diff
@@ -34,19 +87,24 @@ void Start()
     /// <param name="amount">By how many units should the maximum capacity be increased.</param>
     public void IncreaseCapacity(int amount = 1)
```

Comment:
This will never get called without a parameter as `public void IncreaseCapacity()` will be called instead